configurator/frontend/src/ui/pages/DestinationsPage/partials/DestinationEditor/DestinationEditor.tsx

```diff
@@ -166,7 +170,7 @@ const DestinationEditor = ({
       } catch (errors) {
         // ToDo: check errors count for fields with few validation rules
         tab.errorsCount = errors.errorFields?.length;
-        throw errors;
+        return null;
```

Comment:
I think it would be better to re-throw the errors here, and if they bother you in your component just catch them there.

Reply:
here it triggered Uncaught error in Promise in some cases. I didn't find better way to resolve that

Reply:
I see, this function is not the commonly used one, but it is used just locally in the component. Since it already gives the user feedback it is safe to leave it as is.